lib/generators/draft/resource/templates/controllers/controller.rb

```diff
@@ -13,24 +13,24 @@ def show
   end
 
   def create
-    @the_<%= singular_table_name.underscore %> = <%= class_name.singularize %>.new
+    the_<%= singular_table_name.underscore %> = <%= class_name.singularize %>.new
```

Comment:
I think that [`singular_table_name`](https://api.rubyonrails.org/classes/Rails/Generators/NamedBase.html#method-i-singular_table_name) and `plural_table_name` should already be camel cased.

They both start from the file name, which Rails [underscores already](https://github.com/rails/rails/blob/fbe2433be6e052a1acac63c7faf287c52ed3c5ba/railties/lib/rails/generators/named_base.rb#L177).

But that isn't really the point of this PR so I don't think it's something you need to fix now.

Reply:
@jelaniwoods I will make an issue for us to look more specifically at`singular_table_name` and `plural_table_name` and see how they function with the draft generators investigate where we can rebase.